specification/src/main/asciidoc/platform/TransactionManagement.adoc

```diff
@@ -235,8 +235,8 @@ commit the transaction.
 
 A Jakarta EE product must provide a transaction manager that is capable of
 coordinating two-phase commit operations across multiple XA-capable JDBC
-databases. If a JDBC driver supports the Jakarta Transactions API’s XA
-interfaces (in the _jakarta.transaction.xa_ package), then the Jakarta EE product
+databases. If a JDBC driver supports the Java Transaction API’s XA
```

Comment:
This should be Java SE actually (https://docs.oracle.com/javase/8/docs/api/javax/transaction/xa/XAResource.html)

Reply:
The original document had Java Transaction API's XA interfaces
https://github.com/eclipse-ee4j/jakartaee-platform/blame/dbb1841d0a7cabead8eb6060db23b0d8c7ffedc8/specification/src/main/asciidoc/platform/TransactionManagement.adoc#L246

Should we change it to Java SE API's XA interfaces or just Java SE XA interfaces.

Reply:
I think it's a relic. The Java expression of the XA API used to be in Java EE (and it may well have been in Java SE at the same time). Now the Java version of XA is just in SE. I think "Java SE API's XA interfaces" is probably OK. Thanks, Tom

Reply:
Done. Changed to "Java SE API's XA interfaces".